src/Tests/HelixTasks/SDKCustomCreateXUnitWorkItemsWithTestExclusion.cs

```diff
@@ -72,7 +79,14 @@ public override bool Execute()
         /// <returns></returns>
         private async Task ExecuteAsync()
         {
-            XUnitWorkItems = (await Task.WhenAll(XUnitProjects.Select(PrepareWorkItem)))
+            var testExclusion = File.ReadAllLines(TestProjectExclusionListPath)
+                .Select(l => l.Trim())
+                .Where(l => !string.IsNullOrWhiteSpace(l));
+
+            var xUnitProjectsCanRunInHelix
+                = XUnitProjects.Where(p => !testExclusion.Any(p.ItemSpec.Contains));
```

Comment:
There is no easy reliable way to do that other than regex in both powershell and bash. It is better to keep the logic in sync, since overlap or missing test projects are worse than accidentally run on host.